### ISSUE TITLE
Show users unnecessary manual subscriptions

### ISSUE THREAD

**Issue by rossjrw** [croque-scp/notifier#51]
Some users subscribe to threads that they don't need to, as they're already covered by the automatic subscription rules. It would be nice to show them a list of such subscriptions such that they can remove them if they choose.

**Comment by rossjrw** [comment]
Similarly to #79, creating this list will not be possible after #85 because I won't be storing users' automatic subscriptions except for those that are actively emitting a notification, and therefore won't be able to compare that list against the manual subscriptions.

That being said, instead of adding a straight-up list to the digest, I can show whether a notification was a match for multiple contexts. If so, and if one of them was a manual subscription, I can advice the user that the manual subscription is unnecessary on a per-notification basis.

**Comment by rossjrw** [comment]
As part of #85 it's looking like (not set in stone yet) the database is going to emit a set of bools per notification indicating which reasons it matched for it to be emitted. I'll be modifying the digester to change the text in the digest based on the presence of one of these flags. I won't be actioning this issue in #85, though, I'll leave it as close to the extant behaviour as possible.

Later I can action this issue specifically by adding special cases for when multiple flags are set on a notification.
